core/dbt/parser/schema_test_builders.py

```diff
@@ -41,14 +41,25 @@ def get_nice_schema_test_name(
     clean_flat_args = [re.sub('[^0-9a-zA-Z_]+', '_', arg) for arg in flat_args]
     unique = "__".join(clean_flat_args)
 
+    # for the file path + alias, the full name must be <64 characters.
+    # if the path is too long, include the first 30 identifying chars plus
+    # a 32-character hash of the contents
+
     cutoff = 32
     if len(unique) <= cutoff:
         label = unique
     else:
         label = hashlib.md5(unique.encode('utf-8')).hexdigest()
 
-    filename = '{}_{}_{}'.format(test_type, test_name, label)
-    name = '{}_{}_{}'.format(test_type, test_name, unique)
+    cutoff = 30
+    test_full_name = '{}_{}'.format(test_type, test_name)
+    if len(test_full_name) <= cutoff:
+        test_prenom = test_full_name
```

Comment:
let's uh... use english.... for variable names :)

Reply:
heh. I decided to reimplement this logic anyway, since it was getting a little confused